galaxykit/command.py

```diff
@@ -452,7 +462,10 @@ def main():
             "username": args.auth_username,
             "password": args.auth_password,
         }
-    client = GalaxyClient(args.server, creds, https_verify=https_verify)
+
+    # greet is here only for test if application is running and without the server
+    if args.kind != "greet":
+        client = GalaxyClient(args.server, creds, https_verify=https_verify)
```

Comment:
this is the part we need to work .. so we can't skip it here if galaxykit greet is supposed to be used for tests